spec.txt

```diff
@@ -722,57 +722,64 @@ recognized as entity references either:
 ````````````````````````````````
 
 
-Entity and numeric character references are recognized in any
-context besides code spans or code blocks, including
-URLs, [link titles], and [fenced code block][] [info strings]:
+Entity and numeric character references are treated as literal
+text in code spans and code blocks:
 
 ```````````````````````````````` example
-<a href="&ouml;&ouml;.html">
+`f&ouml;&ouml;`
 .
-<a href="&ouml;&ouml;.html">
+<p><code>f&amp;ouml;&amp;ouml;</code></p>
 ````````````````````````````````
 
 
 ```````````````````````````````` example
-[foo](/f&ouml;&ouml; "f&ouml;&ouml;")
+    f&ouml;f&ouml;
 .
-<p><a href="/f%C3%B6%C3%B6" title="föö">foo</a></p>
+<pre><code>f&amp;ouml;f&amp;ouml;
+</code></pre>
 ````````````````````````````````
 
 
+
+Entity and numeric character references are left uninterpreted
+in HTML blocks and raw HTML:
```

Comment:
```suggestion
Entity and numeric character references are not interpreted
in HTML blocks and raw HTML:
```

Personally I think this suggestion is clearer, but I don’t have strong opinions on it.